src/import/duplicate_detect.rs

```diff
@@ -490,6 +498,11 @@ mod tests {
         ];
         let snapshot = storage.get_snapshot();
         let detector = DuplicateDetector::new(snapshot, b"0".to_vec(), None, 13, false).unwrap();
+        check_duplicate_data(detector, expected_kvs.clone());
+
+        // in fact lightning will not set min_commit_ts
+        let snapshot = storage.get_snapshot();
+        let detector = DuplicateDetector::new(snapshot, b"0".to_vec(), None, 0, false).unwrap();
```

Comment:
I did not understand how this test will test the new logic, as it seems no WriteType::Delete in test case.